Client/javascripts/es6/script.js

```diff
@@ -173,6 +173,36 @@ function forgotPassword() {
 	}
 }
 
+// eslint-disable-next-line
+function viewEntries() {
+	document.querySelector('#dashboard').style.display = 'none';
+	document.querySelector('#index').style.display = 'block';
+	fetch(`${baseUrl}/entries`, {
+		method: 'GET',
+		headers: {
+			'Content-Type': 'application/json',
+			token: localStorage.getItem('diary_token'),
+		},
+	})
+		.then(res => res.json())
+		.then((data) => {
```

Comment:
You can destructure `data` right from here by doing:

`.then(({ data }) => {})`

So line 191 will become `data.map`.